OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseTabsDashboardFragment.kt

```diff
@@ -813,7 +813,7 @@ class CourseTabsDashboardFragment : BaseFragment() {
             )
```

Comment:
Please check all the occurrences of `analyticsRegistry.trackScreenView` in this file and also formate it accordingly.

Reply:
I updated one of the calls but couldn't find anything else to format.

Reply:
The code is already formatted. 👍🏻